flink-tests/src/test/java/org/apache/flink/test/checkpointing/ResumeCheckpointManuallyITCase.java

```diff
@@ -94,71 +94,63 @@ public static Object[] parameters() {
     @Test
     public void testExternalizedIncrementalRocksDBCheckpointsStandalone() throws Exception {
         final File checkpointDir = temporaryFolder.newFolder();
+        StateBackend stateBackend = createRocksDBStateBackend(checkpointDir, true);
         testExternalizedCheckpoints(
-                checkpointDir,
-                null,
-                createRocksDBStateBackend(checkpointDir, true),
-                false,
-                restoreMode);
+                checkpointDir, null, stateBackend, stateBackend, false, restoreMode);
```

Comment:
Actually, we don't need to change these existing tests.
We can introduce another static method below to keep previous tests stay as before.
~~~java
    private static void testExternalizedCheckpoints(
            File checkpointDir,
            String zooKeeperQuorum,
            boolean localRecovery,
            RestoreMode restoreMode) throws Exception {
        StateBackend stateBackend = createRocksDBStateBackend(checkpointDir, true);
        testExternalizedCheckpoints ...
    }
~~~